### PR TITLE
Issue 17

### PR DESCRIPTION
**Pull request by mroark1m**
Fixes Issue #17

### CODE REVIEW

File: scripts/endpoint_dns_name.sh [hunk 1]
```diff
@@ -1,3 +1,7 @@
 #!/bin/sh
 dns_name=`aws ec2 describe-vpc-endpoints --vpc-endpoint-ids $1|jq -r ".VpcEndpoints[].DnsEntries[0].DnsName"`
+if [ "$dns_name" = "null" ]; then
+  echo dns_name for vpc endpoint is null, one issue which may cause this is an older version of the aws cli on the machine you run terraform from. 1>&2
```
Comment (massdosage) [inline scripts/endpoint_dns_name.sh]: NABD but I usually prefer the args to `echo` to be quoted, I think it reads a bit better. You could also consider vpc->VPC, aws->AWS, cli-CLI, terraform->Terraform.

Above are all minor for something that hopefully only gets printed in unusual circumstances so I'll approve as-is (and well re-approve if you decide you want to make the changes)

Comment (pradeepbhadani) [reply scripts/endpoint_dns_name.sh]: I think similar to https://github.com/ExpediaInc/apiary-metastore-docker/blob/master/files/startup.sh#L62

**Comment by mroark1m** [comment]
@pradeepbhadani @massdosage please re-review.

**Comment by mroark1m** [comment]
@rpoluri @massdosage pls reapprove

**Pull request merged**